index.html

```diff
@@ -10,20 +10,67 @@
 </head>
 
 <body>
-    <section class="dataForm">
-        <!-- display dynamic content -->
+    <!-- Header Section -->
+    <header class="header-section">
+        <div class="nav-header">
+            <p>Awsome Books</p>
+        </div>
+        <nav>
+            <ul>
+                <li class="links" id="bookList">List</li>
+                <li class="links" id="addBook">Add New</li>
+                <li class="links" id="contact">Contact</li>
+            </ul>
+        </nav>
+    </header>
+    <div class="date-section">
+        <p>
+            13 aug 2022
+        </p>
+    </div>
```

Comment:
- [ ] You have carefully followed the wireframe design as closely as possible but you missed the time at the top-right of the page as shown below
### Wireframe
![image](https://user-images.githubusercontent.com/46839548/184483663-d722b1fc-5de1-4462-81ac-1b3932b752da.png)

### Your Design
![image](https://user-images.githubusercontent.com/46839548/184484025-835b1683-bbcf-4a4b-be80-7c7eb0df143f.png)


- [ ] [OPTIONAL] you are hardcoding the date, but you can make make it dynamic by using this handle library called [luxon.js](https://cdnjs.cloudflare.com/ajax/libs/luxon/3.0.1/luxon.min.js)

Reply:
- [ ] As requested by the previous reviewer, make sure that you implement also the time. I'll suggest you use the luxon library. Check [this link](https://moment.github.io/luxon/#/) to see how to use it.